cypress/integration/aboutTheVessel.spec.ts

```diff
@@ -124,9 +114,6 @@ describe("As a beacon owner, I want to submit information about my beacon", () =
         ...tooLongErrorMessageContains
       );
 
-      thenIShouldSeeAnErrorSummaryLinkThatContains(
-        ...tooLongErrorMessageContains
-      );
```

Comment:
Removed duplicated assert statements

Reply:
Great idea!  No use in asserting twice that the error summary link exists, both when Cypress sees it and when Cypress clicks on it.  👌🏻